src/en/tercoscans/src/eu/kanade/tachiyomi/extension/en/tercoscans/TercoScans.kt

```diff
@@ -0,0 +1,35 @@
+package eu.kanade.tachiyomi.extension.en.tercoscans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class TercoScans : MangaThemesia(
+    "Terco Scans",
+    "https://tecnoxcomics.xyz",
+    "en",
+) {
+    // I suppose that Terco Scans is also maintained by someone of Eros Scans
+    override val client = super.client.newBuilder()
```

Comment:
J2K does not use the Cloudflare client by default.
```suggestion
    override val client = network.cloudflareClient.newBuilder()
```